samples/drivers/watchdog/boards/yd_esp32.conf

```diff
@@ -0,0 +1 @@
+CONFIG_HEAP_MEM_POOL_SIZE=256
```

Comment:
I think this .conf is not needed anymore (any esp32 board).

Reply:
Done. The `.conf` has been removed.

Sample output after removal:

![yd_esp32_watchdog_sample](https://github.com/zephyrproject-rtos/zephyr/assets/103332557/97087a0d-c717-4f54-9c79-847328ec3651)